docs/spaces.rst

```diff
@@ -84,7 +84,11 @@ that do not generate *standardized* coordinate spaces:
     including the ``fsnative`` space will instruct fMRIPrep to sample the
     original BOLD data onto FreeSurfer's reconstructed surfaces for this
     individual.
-  * **Additional nonstandard spaces** which are being discussed
+  * ``func``, ``bold``, ``run``, ``boldref`` or ``sbref`` can be used to
```

Comment:
At this point all of them are equivalent. Once we make the corresponding decisions in the context of #1604, then we can refine the list.

That said, i) I agree with your inclination; and ii) when you have time -and probably this can wait until after OHBM-, let's dive into a concrete definition of nonstandard spaces and keywords.